cli/src/template_builder.rs

```diff
@@ -1223,10 +1224,19 @@ fn builtin_timestamp_methods<'a, L: TemplateLanguage<'a> + ?Sized>(
             let [format_node] = function.expect_exact_arguments()?;
             let format =
                 template_parser::expect_string_literal_with(format_node, |format, span| {
-                    time_util::FormattingItems::parse(format)
-                        .ok_or_else(|| TemplateParseError::expression("Invalid time format", span))
+                    // We format a dummy timestamp in order to validate the format string.
+                    match strtime::format(
+                        format,
+                        &jiff::civil::date(2020, 12, 20)
+                            .at(6, 37, 0, 0)
+                            .to_zoned(jiff::tz::TimeZone::UTC)
+                            .unwrap(),
+                    ) {
```

Comment:
this would be the perfect place to add the date of the first commit in the private Mercurial repo as an easter egg cc @martinvonz

Reply:
This date wasn't quite so random either... 😉

Reply:
I'm pretty sure its the initial Git import :-).

Reply:
Indeed: 25caaa0081223508c45b1dfe9feeac60c925eb45 ;-)

Reply:
Guess it might make sense to do 6b1427cb46a68315d66fa93ea1ebf54b0ecc4b0a if not the hg repo though.

Reply:
Just construct zeroed `backend::Timestamp` and convert it to `Zoned`?